### DISCUSSION TITLE
helm部署分布式milvus时，values.yaml中querynode 如何配置TopologySpreadConstraints

### DISCUSSION THREAD

**Discussion by gaoyuan5251** [milvus-io/milvus#38613] [Q&A and General discussion] [answered]
我想给querynode的配置，增加一个调度配置TopologySpreadConstraints，不想在templates中配置，如何在values.yaml中querynode:下添加呢？我试了好多种方式都不行。
<img width="513" alt="image" src="https://github.com/user-attachments/assets/41cafe72-680d-4fbe-8162-63253d8f6e71" />
<img width="1149" alt="image" src="https://github.com/user-attachments/assets/0bb93087-b1c2-4028-b78d-eef5e5306e7f" />

**Comment by yhmo** [comment]
Wait @LoveEachDay comment.

**Reply by LoveEachDay** [reply]
@gaoyuan5251 The configuration of `topologySpreadConstraints` doesn't support in milvus-helm chart. @yellow-shine Would you help to add this feature in milvus-helm chart?

**Reply by gaoyuan5251** [reply, marked as answer]
> @gaoyuan5251 The configuration of `topologySpreadConstraints` doesn't support in milvus-helm chart. @yellow-shine Would you help to add this feature in milvus-helm chart?

@LoveEachDay @yellow-shine Got it. Thanks for the support. There is another LB configuration that doesn't seem to work（milvus v2.4.17 ）
<img width="1122" alt="image" src="https://github.com/user-attachments/assets/7a2658f0-4384-460c-a5d1-2d0148b2a501" />

**Comment by yellow-shine** [comment]
https://github.com/zilliztech/milvus-helm/pull/167 , this pr will support  topologySpreadConstraints
